libs/cart/src/drivers/interfaces/transformers/cart-transformer.interface.ts

```diff
@@ -0,0 +1,16 @@
+import { DaffCart } from '../../../models/cart';
+import { InjectionToken } from '@angular/core';
+
+/**
+ * An interface that must be implemented by cart transformer services.
+ * 
+ * This transformer is responsible for transforming the returned
+ * cart from a given platform into a Generic DaffCart.
+ */
+export interface DaffCartTransformerInterface<RES, T extends DaffCart> {
+  transform(response: RES): T;
```

Comment:
I kinda like how you've been doing `transform(response: any): T;`, because it cuts down on how much boiler plate you have when you implement the interface. What do you think?